frame/staking/src/lib.rs

```diff
@@ -2066,14 +1946,11 @@ decl_module! {
 		///         ErasStakersClipped, ErasRewardPoints, ErasValidatorPrefs (8 items)
 		/// - Read Each: Bonded, Ledger, Payee, Locks, System Account (5 items)
 		/// - Write Each: System Account, Locks, Ledger (3 items)
+		///
+		///   NOTE: weights are assuming that payouts are made to dead controller accounts.
```

Comment:
@shawntabrizi is this known and intentional (this behaviour also existed in the previous weights)?

Reply:
Yes, this is how the benchmarks have been, since it is worst condition.